### ISSUE TITLE
[JavaScript Error: "Unexpected file upload status 400 (1/ )"] Error: Data size larger than expected | The body of your POST request is not well-formed multipart/form-data

### ISSUE THREAD

**Issue by uniuuu** [uniuuu/zotprime#6]
```

zotprime-zotprime-dataserver-1     | 172.23.0.1 - - [06/May/2023:21:41:07 +0700] "POST /users/1/items/5NE9EXJD/file HTTP/1.1" 200 2303 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Zotero/6.0.27.SOURCE.2f0d41c0c"
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | API: PostPolicyBucket(bucket=zotero)
zotprime-zotprime-minio-1          | Time: 21:41:07 WIB 05/06/2023
zotprime-zotprime-minio-1          | RequestID: 175C94E3E8EFD820
zotprime-zotprime-minio-1          | RemoteHost: 172.23.0.11
zotprime-zotprime-minio-1          | UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Zotero/6.0.27.SOURCE.2f0d41c0c
zotprime-zotprime-minio-1          | Error: Data size larger than expected
zotprime-zotprime-minio-1          |        1: cmd/handler-utils.go:252:cmd.validateFormFieldSize()
zotprime-zotprime-minio-1          |        2: cmd/handler-utils.go:273:cmd.extractPostPolicyFormValues()
zotprime-zotprime-minio-1          |        3: cmd/bucket-handlers.go:533:cmd.objectAPIHandlers.PostPolicyBucketHandler()
zotprime-zotprime-minio-1          |        4: cmd/api-router.go:109:cmd.objectAPIHandlers.PostPolicyBucketHandler-fm()
zotprime-zotprime-minio-1          |        5: pkg/handlers/http-tracer.go:144:handlers.TraceReqHandlerFunc.func1()
zotprime-zotprime-minio-1          |        6: net/http/server.go:1964:http.HandlerFunc.ServeHTTP()
zotprime-zotprime-minio-1          | [REQUEST objectAPIHandlers.PostPolicyBucketHandler-fm] [168338406.731633] [2023-05-06 21:41:07 +0700]
zotprime-zotprime-minio-1          | POST /zotero/
zotprime-zotprime-minio-1          | Host: localhost:8082
zotprime-zotprime-minio-1          | Accept-Language: en-US,en;q=0.5
zotprime-zotprime-minio-1          | X-Zotero-Version: 6.0.27.SOURCE.2f0d41c0c
zotprime-zotprime-minio-1          | User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Zotero/6.0.27.SOURCE.2f0d41c0c
zotprime-zotprime-minio-1          | Accept: */*
zotprime-zotprime-minio-1          | Accept-Encoding: gzip, deflate
zotprime-zotprime-minio-1          | Content-Type: multipart/form-data; boundary=---------------------------492b2b2f6f9326b09769e20b49e927ba
zotprime-zotprime-minio-1          | Content-Length: 1800553
zotprime-zotprime-minio-1          | Connection: keep-alive
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | <BODY>
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | [RESPONSE] [168338406.731633] [2023-05-06 21:41:07 +0700]
zotprime-zotprime-minio-1          | 400 Bad Request
zotprime-zotprime-minio-1          | Vary: Origin
zotprime-zotprime-minio-1          | X-Xss-Protection: 1; mode=block
zotprime-zotprime-minio-1          | Content-Security-Policy: block-all-mixed-content
zotprime-zotprime-minio-1          | X-Amz-Request-Id: 175C94E3E8EFD820
zotprime-zotprime-minio-1          | X-Minio-Deployment-Id: 190c04e7-b266-46bb-8866-bc219aa660db
zotprime-zotprime-minio-1          | Server: Minio/RELEASE.2019-01-31T00-31-19Z
zotprime-zotprime-minio-1          | Accept-Ranges: bytes
zotprime-zotprime-minio-1          | Content-Type: application/xml
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | <?xml version="1.0" encoding="UTF-8"?>
zotprime-zotprime-minio-1          | <Error><Code>MalformedPOSTRequest</Code><Message>The body of your POST request is not well-formed multipart/form-data.</Message><Resource>/zotero/</Resource><RequestId>175C94E3E8EFD820</RequestId><HostId>3L137</HostId></Error>
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | 
zotprime-zotprime-minio-1          | API: PostPolicyBucket(bucket=zotero)
zotprime-zotprime-minio-1          | Time: 21:41:07 WIB 05/06/2023
zotprime-zotprime-minio-1          | RequestID: 175C94E3E8EFD820
zotprime-zotprime-minio-1          | RemoteHost: 172.23.0.11
zotprime-zotprime-minio-1          | UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Zotero/6.0.27.SOURCE.2f0d41c0c
zotprime-zotprime-minio-1          | Error: Data size larger than expected
zotprime-zotprime-minio-1          |        1: cmd/bucket-handlers.go:535:cmd.objectAPIHandlers.PostPolicyBucketHandler()
zotprime-zotprime-minio-1          |        2: cmd/api-router.go:109:cmd.objectAPIHandlers.PostPolicyBucketHandler-fm()
zotprime-zotprime-minio-1          |        3: pkg/handlers/http-tracer.go:144:handlers.TraceReqHandlerFunc.func1()
zotprime-zotprime-minio-1          |        4: net/http/server.go:1964:http.HandlerFunc.ServeHTTP()

```

```
[JavaScript Error: "Upload request 1/3AWQJRTH failed"]

[JavaScript Error: "Unix error 2 during operation stat on file /home/qubit/Zotero/storage/3AWQJRTH/0976-500x.85940.pdf (No such file or directory)" {file: "(unknown module)" line: 0}]

[JavaScript Error: "Unexpected file upload status 400 (1/3AWQJRTH)"]

[JavaScript Error: "<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>MalformedPOSTRequest</Code><Message>The body of your POST request is not well-formed multipart/form-data.</Message><Resource>/zotero/</Resource><RequestId>175CC33957B76DE8</RequestId><HostId>3L137</HostId></Error>" {file: "chrome://zotero/content/xpcom/storage/zfs.js" line: 726}]
Zotero.Sync.Storage.Mode.ZFS.prototype._uploadFile<@chrome://zotero/content/xpcom/storage/zfs.js:726:9
tryCatcher@resource://zotero/loader.jsm -> resource://zotero/bluebird/util.js:16:16
module.exports/PromiseSpawn.prototype._promiseRejected@resource://zotero/loader.jsm -> resource://zotero/bluebird/generators.js:106:18
module.exports/Promise.prototype._settlePromise@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:611:17
module.exports/Promise.prototype._settlePromise0@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:649:5
module.exports/Promise.prototype._settlePromises@resource://zotero/loader.jsm -> resource://zotero/bluebird/promise.js:725:13
_drainQueueStep@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:93:9
_drainQueue@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:86:9
Async.prototype._drainQueues@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:102:5
Async/this.drainQueues@resource://zotero/loader.jsm -> resource://zotero/bluebird/async.js:15:9

[JavaScript Error: "Upload request 1/3AWQJRTH failed"]

[JavaScript Error: "A file sync error occurred. Please try syncing again.

If you receive this message repeatedly, restart Zotero and/or your computer and try again. If you continue to receive the message, submit an error report and post the Report ID to a new thread in the Zotero Forums." {file: "chrome://zotero/content/xpcom/storage/zfs.js" line: 727}]
```

**Comment by uniuuu** [comment]
Fixed by d04ee9db490dea68f090af59a91e3eea5f40211f

**Comment by uniuuu** [comment]
https://github.com/minio/minio/issues/13518
